docker/Dockerfile

```
@@ -0,0 +1,31 @@
+## First stage: build the application
+FROM golang:1.13.0-alpine AS builder
+ARG GOARCH
+ARG GOARM
+
+WORKDIR /go/src/github.com/CESARBR/knot-babeltower
+COPY . .
+
+# Install glide dependency manager
+RUN apk update \
+    && apk add make git curl \
+    && curl https://glide.sh/get | sh
```

What do you think about adding the `--update` (_see [this](https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management#Update_the_Package_list) for details_) and the `--no-cache` flags (_see [this](https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md) and [this](https://stackoverflow.com/questions/49118579/alpine-dockerfile-advantages-of-no-cache-vs-rm-var-cache-apk) for more details_)? Given that we were discussing image sizes, it seems appropriate! :smile: 

Also, it seems to me that these packages are build dependencies and will no longer be needed after (I might be wrong about **glide**, though). If so, using the `--virtual` flag (_see [this](https://stackoverflow.com/questions/46221063/what-is-build-deps-for-apk-add-virtual-command ) for details_) and deleting the named package will further reduce the image size. 

**Obs.**: add `RUN apk del .build-deps` after doing all jobs which require `make, git, curl, glide`.

```suggestion
RUN apk --update add --no-cache --virtual .build-deps \
    make \
    git \ 
    curl \
    && curl https://glide.sh/get | sh
```

---

@lcbm thanks for your suggestion (and the references). Indeed, seems appropriate to use `--update` and `--no-cache` even though the changes caused no significant effect on the image size, the multi-stage already reduce it to approximately 18MB. The intermediate image, created in the first stage, continues to be larger but this isn't a problem since it isn't uploaded to the Docker hub registry.